scripts/travis/cleanup.js

```diff
@@ -84,20 +56,51 @@ const getCurrentRevs = (db, response) => {
   return db.allDocs({ keys: ids });
 };
 
+const queryReleases = (db, daysToKeep) => {
+  return db
+    .query('builds/releases', {
+      startkey: [ 'branch', 'medic', 'medic' ],
+      endkey: [ 'branch', 'medic', 'medic', getEndDate(daysToKeep) ],
+      limit: MAX_BUILDS_TO_DELETE
+    })
+    .then(response => getCurrentRevs(db, response));
+};
+
+const getTestingBuilds = db => {
```

Comment:
I tried that, but then I felt uncomfortable with the db being hardcoded in the `get` function and then hardcoded the `builds` function being passed to the `delete` function. The problem was with this specified in two places it'd be easy to mix and match so you're getting from one DB and deleting from another. This would fail fairly obviously but I didn't like it.

Ultimately I decided to inline these one line `get` functions and pull out some common parts in other ways. See what you think!